src/checks/noDependencyProblems/noUnusedDependencies.ts

```diff
@@ -88,6 +121,7 @@ export default async function noUnusedAndMissingDependencies() {
       `);
     }
 
-    throw new Error(messages.join('\n\n'));
+    // TODO: Remove the condition once the issue 789 is fixed in depcheck
+    if (messages.length) throw new Error(messages.join('\n\n'));
```

Comment:
can we avoid this extra section at the bottom? makes it harder to remember to remove this too

Reply:
we removed the TODO instead